test/test_train_cifar.py

```diff
@@ -221,19 +221,24 @@ def test_loop_fn(model, loader, device, context):
       xm.xla_replication_devices(devices)) if len(devices) > 1 else 1
   num_training_steps_per_epoch = train_dataset_len // (
       FLAGS.batch_size * num_devices)
+  max_accuracy = 0.0
   for epoch in range(1, FLAGS.num_epochs + 1):
     model_parallel(train_loop_fn, train_loader)
     accuracies = model_parallel(test_loop_fn, test_loader)
     accuracy = mean(accuracies)
-    print("Epoch: {}, Mean Accuracy: {:.2f}%".format(epoch, accuracy))
+    if accuracy > max_accuracy:
+      print('New best accuracy:')
```

Comment:
I'm not sure that print belongs there.

Reply:
Removed the 'new best accuracy' print statements from the 5 tests and switched to more succinct way of updating max_accuracy